app/cms/utils/fetch-discourse-api.tsx

```diff
@@ -50,28 +59,146 @@ export interface Topic {
   category_id: number
   pinned_globally: boolean
   featured_link: string
-  posters: Poster[]
+  posters: any
+}
+
+interface CategoryResponse {
+  category: Category
 }
 
-export interface Poster {
-  extras: string
+interface Category {
```

Comment:
Oof this must have been painful. I wonder if there is a better way to do this?

Reply:
We can create an enum style mapping in our code as long as the category ID doesn't change!